### DISCUSSION TITLE
Subsystem name for carbon metabolism

### DISCUSSION THREAD

**Discussion by mihai-sysbio** [SysBioChalmers/yeast-GEM#335] [Q&A]
I'd like to know more about the background of the naming for `alternate carbon metabolism`, and the absence of a `carbon metabolism` subsystem.

**Comment by IVANDOMENZAIN** [comment]
`Carbon metabolism` is a subsystem in KEGG that encompasses many pathways: glycolysis, pentose-phosphate pathway, TCA cycle, glyoxylate shunt, gluconeogenesis, anapletoric pathways and even methane metabolism (for the case of methanogen microorganisms). As all these smaller pathways are also associated to subsystems identifiers in KEGG, it is more informative for a user to associate reactions in the model to these smaller, and more recognizable susbsystems. 

Currently, there's no subsystem in KEGG associated to the name `alternate carbon metabolism`. I can see that there are some few reactions in the model associated to such subsystem, however it was not possible for me to track the origin of such annotation.

**Reply by mihai-sysbio** [reply]
Thanks for the explanation @IVANDOMENZAIN.

It sounds like this `alternate carbon metabolism` should be turned into something else. What do you think @edkerk @feiranl @SysBioChalmers/yeast ?

edit: ~I'm also converting this discussion into an issue, since it is tied to another issue in https://github.com/MetabolicAtlas/data-files/issues/35.~ Let's conclude the discussion first.
